### ISSUE TITLE
Add additionalOptions test example

### ISSUE THREAD

**Issue by afontcu** [testing-library/vue-testing-library#49]
Adding a test example for `additionalOptions` feature (#48 #47) would both (1) show how to use it and (2) increase code coverage.

I guess using `vue-i18n` as a example would be great.

**Comment by makeupsomething** [comment]
I can work on this as I actually implemented myself today. Not sure if I did it correctly or not though.

```js
import Vue from 'vue'
import Vuei18n from 'vue-i18n'
import ja from '@/translations/ja'

Vue.use(Vuei18n)

const i18n = new Vuei18n({
    locale: 'en',
    messages: {
        ja,
    },
})

import {cleanup, render, fireEvent} from '@testing-library/vue'

afterEach(cleanup)

test('can navigate to signup page from welcome page', async () => {
    const {queryByTestId, getByText, debug} = render(
        App,
        {
            routes,
            store,
            i18n,
        },
        vue => vue.use(Vuei18n),
    )
})
```

If theres a better way let me know and I can tidy this up and add to the tests

**Comment by afontcu** [comment]
Hi! Thank you for taking this :)

Notice how, in your snippet, you are using an external Vue instance to register i18n, so you are not actually using the instance exposed by Vue Testing Library. Notice how, in issue https://github.com/testing-library/vue-testing-library/issues/47, the example suggests returnint `{ i18n }` from the callback function. This is the feature we added on #48.

Long story short: You shouldn't import and use external Vue, but rather use the Vue instance provided in the callback function. Then, create the config there, and return the desired object keys.

If you want, open up a PR and we can work from there :)

Thanks!

**Comment by makeupsomething** [comment]
Hi! 

Yes, I was sure that I should not be importing the external Vue to create the i18n instance but could not figure out how to create it any other way. I had not seen #47 at the time but that is clearly the correct way to create it. 

Il adjust things and create a PR over the weekend.

**Comment by afontcu** [comment]
Closed in #58
